web-components/navdots/navdots.svelte

```diff
@@ -36,26 +38,41 @@
       </li>
     {/each}
 
-    <li aria-hidden="true" class="active-dot" />
+    <li
+      aria-hidden="true"
+      class="active-dot"
+      class:no-animate={!animateSlide}
+    />
   </ol>
 </nav>
 
 <style lang="scss">
-  :root {
-    --dot-size: 8px;
-    --dot-spacing: 10px;
-    --dot-vertical-margin: 1px;
-    --current-dot: 0;
-    --transition-duration: 0.2s;
-    --transition-easing: ease-in-out;
-
+  .leo-navdots {
+    --dot-size: var(--leo-navdots-size, 8px);
     // Expanded dot grows to fill half spacing in each direction.
-    --expanded-dot-size: calc(var(--dot-size) + var(--dot-spacing));
+    --expanded-dot-size: var(
```

Comment:
Updated all the CSS properties to use the same pattern as everything else

Reply:
For personal edification: what pattern are you referring to?